aws/ec2/deploy/home/ec2-user/snappydata/zeppelin-setup.sh

```diff
@@ -142,7 +141,11 @@ else
   # TODO If user has made any changes to the interpreter config, those may be lost.
   # Start and stop the Zeppelin daemon
   sh "${ZEP_DIR}/bin/zeppelin-daemon.sh" start
-  sleep 2
+   while ! test -f  "${ZEP_DIR}/conf/interpreter.json" ; do
+      sleep 15
+      echo "Configuring Snappydata Interpreter..."
```

Comment:
Please move this echo outside while loop, otherwise it'll keep printing it until the check succeeds. Or remove it.